posts/jhipster-5-release.md

```diff
@@ -0,0 +1,218 @@
+---
+authors:
+- Kenneth Hegeland, Theo Lebrun, Raphael Brugier
```

Comment:
List authors on separate lines (this is a YAML list)

Reply:
👍